packages/portfolio-contract/test/rebalance.test.ts

```diff
@@ -656,7 +656,7 @@ test('solver differentiates cheapest vs. fastest', async t => {
         src: '@agoric',
         dest: '@External' as AssetPlaceRef,
         transfer: 'cheap' as TransferProtocol,
-        variableFeeBps: 5,
+        variableFeeBps: 2,
```

Comment:
Since this test is being marked as failing, let's maintain its contents.
```suggestion
        variableFeeBps: 5,
```